### DISCUSSION TITLE
Option to dowload backups quietly removed

### DISCUSSION THREAD

**Discussion by hmnd** [supabase/supabase#17932] [General]
In #17010, the button to download a copy of a backup was quietly removed. There was no announcement of this change and there is no replacement offered for this functionality. This seems like a push for users to have to pay for Point-in-Time Recovery, rather than using a downloaded backup to partially restore data themselves.

Supabase's biggest selling point is its openness. Pulling a long-standing feature in order to promote a paid replacement feels like a huge red flag. This makes me afraid to rely on any of Supabase's other features, without also having to maintain my own fallback in case they get quietly removed too. Supabase is supposed to be a better alternative to Firebase from Google, but this is exactly the kind of thing that Google does all the time.

Supabase, please bring back this essential feature and do better by announcing (or better yet, discussing!) deprecation of other features going forward.

cc @kiwicopple

**Comment by leerobert** [comment]
Is there a manual way to download this backup or are we expected now to run our own pgdumps?

**Reply by hmnd** [reply]
As far as I can tell, there is not. The endpoint formerly used for this now returns an empty 400 response.

**Comment by hmnd** [comment]
Looks like this has been reversed with #18070!
